### PR TITLE
misspelling

### PR DESCRIPTION
**Pull request by franzholz**
In British English you write "behaviour".

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
Does "en" not mean American English, and `en-gb" mean british?

**Comment by franzholz** [comment]
"en_us" is American English.
"en" should stand for British English.

**Comment by GrahamCampbell** [comment]
Ok, cool. :+1: then.

**Comment by sebastianbergmann** [comment]
Thank you for your recent changes. However, to make our work easier, could you please send pull requests for all (English) editions? Thanks!

**Comment by whatthejeff** [comment]
I don't think [`RFC 5646`](https://tools.ietf.org/html/rfc5646) specifies default regions for any language tags. I think "en" just means English if no region is specified.

@sebastianbergmann Do you have a preference between British and American English for the documentation?

**Comment by sebastianbergmann** [comment]
I learned Oxford English in school but am mostly exposed to American English in my professional life. It's hard to say, but I think that I prefer Oxford English.

**Comment by sun** [comment]
Sorry for entering the discussion late.  I'd like to throw in a few data points:

1. The current PHPUnit manual uses American English all over the place.

  Otherwise, as one of many examples, the _["Organizing Tests"](https://phpunit.de/manual/current/en/organizing-tests.html)_ chapter would be spelled _"Organi**s**ing Tests"_.

1. The PHPUnit software application itself exposes options that are in American English, such as `--colors`, not `--colours`.
  _(as also identified in sebastianbergmann/phpunit#1572)_

  Otherwise, the manual would talk about _"colours"_ whereas the user input is _"colors"_, confusing.

1. As Jeff already pointed out, _"en"_ is a generic language identifier without a specified region.  It's neither American English, nor British English, nor Australian English, but just English.

  In the same way _"de"_ includes no region either.  It is neither German as used in Germany (de-de), nor as used in Austria (de-at), nor as used in Switzerland (de-ch), but just German.

1. I'm not aware of a modern software manual that is written in British English.

  It appears that PHPUnit currently follows the norm, American English.

1. While British English and Britain is generally associated with more authority and style, and while it would probably be fun to convert the whole manual and application into British English (just for kicks)… I believe the project is generally short on contributors and everyone's time is much better spent elsewhere.

  Also, in terms of fun, there are much better options, such as [`xx-lolspeak`](https://localize.drupal.org/translate/languages/xx-lolspeak) or [Klingon](http://en.wikipedia.org/wiki/Klingon_language).

In short, PHPUnit clearly appears to use American English, not British English.  I'd recommend to leave it that way, and close this + related issues.

Lastly, the real take-away here from my perspective:  There's no localization in PHPUnit (yet).  But I also do not think that PHPUnit needs to be localized (aside from translating the manual).

**Comment by franzholz** [comment]
I only want to add: Such a discussion would take more time than fixing this issue. There are only a few lines which needs fixing.

**Comment by whatthejeff** [comment]
I discussed this with @sebastianbergmann on gitter. We decided to stick with the current spelling since American English is more ubiquitous in the PHP world.
